docs/book/src/migration/v2vsv3.md

```diff
@@ -2,21 +2,71 @@
 
 This document covers all breaking changes when migrating from v2 to v3.
 
-The details of all changes (breaking or otherwise) can be found in [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases) release notes.
+The details of all changes (breaking or otherwise) can be found in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/releases),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools/releases)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases)
+release notes.
+
+## Common changes
+
+v3 projects use go modules and at least Go 1.15; Dep is no longer supported.
 
 ## Kubebuilder
 
-- A plugin design was introduced to the project. For more info see the [Extensible CLI and Scaffolding Plugins][plugins-phase1-design-doc]
-- The GO supported version was upgraded from 1.13+ to 1.15+
+- Preliminary support for plugins was added. For more info see the [Extensible CLI and Scaffolding Plugins][plugins-phase1-design-doc]
+
+- The `PROJECT` file now has a new layout.  It stores more information about what resources are in use, to better enable plugins to make useful decisions when scaffolding.
+    
+    Furthermore, the PROJECT file itself is now versioned: the `version` field corresponds to the version of the PROJECT file itself, while the `layout` field indicates the scaffolding & primary plugin version in use. 
+    
+- Upgrade of the version `gcr.io/kubebuilder/kube-rbac-proxy` from `0.5.0` to `0.8.0` to address security concerns. The details of all changes can be found in [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy/releases),
```

Comment:
```suggestion
- Upgrade kube-rbac-proxy, an optional component to secure the Prometheus metrics, from `0.5.0` to `0.8.0` to address security concerns. The details of all changes can be found in [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy/releases).
```

Reply:
I would not like to add `an optional component to secure the Prometheus metrics,` because it is scaffold by default and it does not allow any request. I mean, it is not to secure the Prometheus metrics and it was added in the project before the metrics.